pkg/provider/kubernetes/crd/traefikio/v1alpha1/serverstransport.go

```diff
@@ -63,6 +67,14 @@ type ForwardingTimeouts struct {
 	PingTimeout *intstr.IntOrString `json:"pingTimeout,omitempty"`
 }
 
+// +k8s:deepcopy-gen=true
+
+// RootCAs holds the reference to the secret or config map that stores the CA.
+type RootCAs struct {
```

Comment:
As the `ServersTransport.RootCAs` property is an array of root CA, could you please rename this type to `RootCA`?

Reply:
done